SQLBuilder/Universal/Expr/BinaryExpr.php

```diff
@@ -20,7 +20,11 @@ public function __construct($operand, $op, $operand2) {
     }
 
     public function toSql(BaseDriver $driver, ArgumentArray $args) {
-        return $this->operand . ' ' . $this->op . ' ' . $driver->deflate($this->operand2, $args);
+        $column = $this->operand;
+        if ($driver->quoteColumn) {
```

Comment:
I think adding a ColumnName type here could break usage without ColumnName box.

Maybe just add a `is_string` for now to make it backward compatible.